doc/JSON_FLAGS.md

```diff
@@ -206,7 +206,8 @@ Flags used to describe monsters and define their properties and abilities.
 ### Flags
 
 - ```NULL``` Source use only.
-- ```ABSORBS``` Consumes objects it moves over.
+- ```ABSORBS``` Consumes objects it moves over. (Modders use this).
```

Comment:
I'd remove this comment. Modders can and should use any possibility the game offers.

Reply:
I added it because atm this flag is not used in the game and if removed it will break mods.

Reply:
`ABSORBS` is used in the game! You change the line with it: ``if( !is_hallucination() && ( has_flag( MF_ABSORBS ) || has_flag( MF_ABSORBS_SPLITS ) )``.

Reply:
There is no monsters with the flag, the only monster was the shoggoth who now has the new splitting one.

Reply:
Would 'legacy feature' be a better wording?